src/pkg/backup/backup.go

```diff
@@ -82,19 +89,27 @@ func New(
 				model.ServiceTag: selector.PathService().String(),
 			},
 		},
-		CreationTime:    time.Now(),
-		SnapshotID:      snapshotID,
-		DetailsID:       detailsID,
-		Status:          status,
-		Selector:        selector,
-		FailFast:        ee.FailFast,
-		ErrorCount:      errCount,
-		Failure:         failMsg,
-		FailedItems:     ee.Items,
-		SkippedItems:    ee.Skipped,
+
+		Version:    version.Backup,
+		SnapshotID: snapshotID,
+		DetailsID:  detailsID,
+
+		CreationTime: time.Now(),
+		Status:       status,
+
+		Selector: selector,
+		FailFast: errs.FailFast(),
+
+		ErrorCount: errCount,
+		Failure:    failMsg,
+
 		ReadWrites:      rw,
 		StartAndEndTime: se,
-		Version:         version.Backup,
+		SkippedCounts: stats.SkippedCounts{
+			TotalSkippedItems: len(ee.Skipped),
+			SkippedMalware:    malware,
+			OtherSkippedItems: otherSkips,
```

Comment:
Minor nit - do we need to track and report `OtherSkippedItems`? `Total` and `Malware` (and other categories in the future) seem sufficient.

Reply:
I suppose not.